src/scripts/README_allocate_fanet_id.md

```diff
@@ -0,0 +1,66 @@
+# FANET ID Allocator for ESP32
+
+> **Warning:** This tool will clear any user-saved settings on your device. Use with caution.
+
+## Overview
+
+The FANET ID Allocator is a Python-based utility for assigning and writing FANET IDs to ESP32 devices. It automatically generates an NVS (Non-Volatile Storage) binary containing the assigned FANET ID and flashes it to the target ESP32. The tool also checks if a MAC address already has a FANET ID and writes a CSV file for record-keeping.
```

Comment:
It seems like we should also document the expectation that the user creates a PR with the updated CSV following the use of this tool.